src/olympia/tags/migrations/0007_tag_enable_for_random_shelf.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.6 on 2021-08-26 13:28
+
+from django.db import migrations, models
+
+
+def set_tag_default(apps, schema_editor):
+    Tag = apps.get_model('tags', 'Tag')
+    Tag.objects.update(enable_for_random_shelf=True)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tags', '0006_auto_20210813_0941'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='tag',
+            name='enable_for_random_shelf',
+            field=models.BooleanField(default=True, null=True),
+        ),
+        migrations.RunPython(set_tag_default),
+        migrations.AlterField(
+            model_name='tag',
+            name='enable_for_random_shelf',
+            field=models.BooleanField(default=True),
+        ),
+    ]
```

Comment:
The reason we generally make new columns nullable at first and then remove the null in a following push is to handle creations of new model instances with the old code but with the new SQL schema *during the push period*. There is no benefit in trying to do both in the same migration: the potential for failure is the same.

By default, Django behaves exactly in the same way, and would fail in the same way if a tag is added during the push, except the migration it generates is a lot more simple:
```python
migrations.AddField(model_name='tag', enable_for_random_shelf', models.BooleanField(default=True))
```

Produces the following SQL:
```sql
ALTER TABLE `tags` ADD COLUMN `enable_for_random_shelf` bool DEFAULT b'1' NOT NULL;
ALTER TABLE `tags` ALTER COLUMN `enable_for_random_shelf` DROP DEFAULT;
```

For a model like `Tag`, where we now fully control the creations, it's fine to not bother with that, so you could just let Django generate the migration without doing anything custom.

Reply:
I was just a little distrustful of django's ability to backfill the existing records with the new column default.  I'll replace it.